.github/workflows/CI.yml

```diff
@@ -79,9 +79,9 @@ jobs:
     - name: Decrypt OpenEye license
```

Comment:
Nitpick: Might be a better verb to use here than "decrypt", even though that's what's happening on a server somewhere (and this PR would be a good place to update this)